### DISCUSSION TITLE
Bluetooth scale

### DISCUSSION THREAD

**Discussion by thepokky** [Zer0-bit/gaggiuino#322] [Ideas]
Are there plans to support Bluetooth scales like the Felicita Arc?

**Comment by banoz** [comment]
> Are there plans to support Bluetooth scales like the Felicita Arc?

Are there any documents around on the BLE protocol used in the scales?

**Reply by thepokky** [reply]
I use an open source application Beanconqueror and it supports these protocols. I think you could use their codes or the codes they refer to.

https://github.com/graphefruit/Beanconqueror

**Reply by banoz** [reply]
So your idea is to reverse engineer someone's code written in a different programming language? That won't fly for sure. Acaia has to publish their protocol to the public, otherwise there won't be much support for their products. And as it said below, there are no plans right now to support more scale configurations.

**Comment by Zer0-bit** [comment]
No plans, only one scales are and will be officially supported, the integrated ones.
